docs/README.linux

```diff
@@ -115,9 +110,33 @@ Tip: To override the location that Kodi is installed, use PREFIX=<path>.
 For example.
 
     $ make install DESTDIR=$HOME/kodi
+	
+-----------------------------------------------------------------------------
+4.4. Binary addons - compile
+-----------------------------------------------------------------------------
+
+From v14 with commit 4090a5f a new API for binary addons is available.
+You can compile all addons or only specific addons by specifying ADDONS="this,pvr,addon this.audio.encoder"
```

Comment:
I think the last part would be more clear with an example like `ADDONS="pvr.foo pvr.bar audioencoder.baz`